.gemini/config.yaml

```diff
@@ -9,8 +14,8 @@ code_review:
     summary: false         # Disable PR summaries (too noisy)
     include_drafts: true   # Review draft PRs for early feedback
     help: false            # Don't post help message on PR open
-  comment_severity_threshold: MEDIUM
-  max_review_comments: -1  # Unlimited review comments
+  comment_severity_threshold: HIGH  # Only HIGH severity (was MEDIUM)
+  max_review_comments: 10  # Limit to 10 comments (was unlimited)
```

Comment:
@rjmurillo-bot why limit? Let the bot get everything out it wants to say so we can be more efficient and require fewer rounds (which has delay). We want to optimize for velocity here, which would be reducing the number of turns. If we write better code, and that code is aligned with reviewers and standards, then we'll have fewer comments on the PR